src/modules/launcher/Plugins/Microsoft.Plugin.WindowWalker/Components/SearchController.cs

```diff
@@ -97,17 +97,7 @@ private SearchController()
         public async Task UpdateSearchText(string searchText)
         {
             SearchText = searchText;
-            await SyncOpenWindowsWithModelAsync();
-        }
-
-        /// <summary>
-        /// Event handler called when the OpenWindows list changes
-        /// </summary>
-        /// <param name="sender">sending item</param>
-        /// <param name="e">event arg</param>
-        public async void OpenWindowsUpdateHandler(object sender, SearchResultUpdateEventArgs e)
```

Comment:
Was removing this event handler intentional?

Reply:
Yes this was.  This was never being invoked in OpenWindow.cs and would have been redundant with the UpdateSearchText call that is called from the Query method in main.cs